less/styles.less

```diff
@@ -51,8 +62,43 @@ body {
   .justify-content(center);
 }
 
-.hero-unit h1 {
+h1 {
   font-weight: 600;
+  font-size: 43px;
+  text-shadow: rgba(0,0,0,.6) 0 2px 1px;
+}
+
+h2 {
+  font-weight: 300;
+  font-size: 35px;
+  margin-bottom: 25px;
+}
+
+h3 {
+  font-weight: 400;
+  font-size: 26px;
+}
+
+main {
+  .list-with-illust {
+    img {
+      display: block;
+      margin: 40px auto 0 auto;
+    }
+  }
+  ol,
+  ul {
+    font-size: 18px;
+    list-style-type: none;
+    padding-left: 0;
+  }
+}
+
+section {
+  padding: 25px;
+}
+
+.hero-unit h1 {
```

Comment:
Hmm should all these `px` measurements be converted to `rem` too? I think @gesa is the expert on this one so I will defer to her.